src/templates/singleBusinessPage.js

```diff
@@ -0,0 +1,325 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+
+import {
+  Box,
+  Flex,
+  Grid,
+  Heading,
+  Icon,
+  Link,
+  Text,
+  useTheme,
+  Stack,
+} from '@chakra-ui/core';
+
+import { Image, LabeledSection, Layout, SEO } from '../components';
+import { Button } from '../components/Button';
+
+import { useImageForBusiness } from '../utils/business';
+import { useBusinessDetails } from '../hooks/useBusinessDetails';
+
+const SingleBusinessPage = ({ data }) => {
+  const theme = useTheme();
+
+  const business = data.airtableBusinesses.data;
+  const {
+    // name,
```

Comment:
Hey @juanri0s - I left these commented out _mostly_ to let me know that these fields are available and yet unused as I'm getting this implementation done.  They should definitely get cleaned up before this gets merged!

Reply:
yupp that makes sense, was just wondering!

In your most recent push, we get a warning for 

`'id' is assigned a value but never used` since you use `recordId` now